acmw-db-app/components/AttendanceForm.jsx

```diff
@@ -0,0 +1,59 @@
+import SelectInput from '../components/FormComponents/SelectInput'
+import TextField from '../components/FormComponents/TextField'
+import SubmitButton from '../components/FormComponents/SubmitButton'
+import styles from '../styles/components/AttendanceForm.module.css'
+import {useState} from 'react'
+import Head from 'next/head'
+
+const AttendanceForm = (props) => {
+
+    const [eventCode, setEventCode] = useState("")
+
+    const yearOptions = ["First", "Second", "Third", "Fourth", "Fifth+", "Graduate or Phd student :)"]
+    const listServOptions = ["I am already on the list :)", "Yes, please!", "Nope, thank you"]
+
+    const handleSubmit = () => {
+        // If event code is valid, add a database entry, and display "Submitted successfully" message
+        // Else if the event code is not valid, display "Invalid event code" message
+
+        // This would be changed to pull from a database of event codes and see if there is a match (maybe?)
+        if (eventCode === "007") {
```

Comment:
Do we also need to look at the other fields (maybe at the very least check if they're not empty)?

Reply:
Good catch--added some more validation checks